style.css

```diff
@@ -541,3 +545,231 @@ input::placeholder {
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+  }
+
+  .header-links {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    left: 0;
+    top: 0;
+    flex-direction: row;
+    text-align: center;
+    height: 40px;
+    padding: 10px 30px;
+  }
+
+  .header-link {
+    color: #172b4d;
+    text-decoration: none;
+    margin: 0 35px;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 16px;
+  }
+
+  .header-link:hover {
+    color: #36b37f;
+  }
+
+  .main-text {
+    padding: 0;
+    margin: 0;
+    min-width: 400px;
+  }
+
+  .works {
+    margin-bottom: 50px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .main-works {
+    font-family: 'Inter', sans-serif;
+    font-weight: 701;
+    text-align: right;
+    color: #172b4d;
+    font-size: 41px;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    z-index: 100;
+  }
+
+  .line-image {
+    margin-left: 95px;
+  }
+
+  .works .works-text {
+    margin-left: 95px;
+    text-align: center;
+    white-space: nowrap;
+  }
+
+  .cards {
+    display: grid;
+    grid-template-columns: repeat(3, 0.21fr);
+    padding: 0.1px;
+    padding-top: 5em;
+    gap: 25px;
+  }
+
+  .about {
+    margin-top: 101px;
+    background-repeat: no-repeat;
+    background-size: contain;
+    margin-bottom: 10px;
+  }
+
+  .about-bg {
+    position: absolute;
+    display: grid;
+    background-image:
+      url("images/illustration-about-me-2-desktop.png"),
+      url("images/illustration-about-me-desktop.png");
+    background-position: 0 22%, 100% 0;
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 100%;
+  }
+
+  .about-card {
+    background-color: #ebf0ee;
+    box-sizing: border-box;
+    border: 1px solid #c1c7d0;
+    border-radius: 8px;
+    padding: 16px;
+    width: 280px;
+    margin: 25px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-left: auto;
+    margin-right: auto;
+    height: 316px;
+  }
+
+  .long-text {
+    margin-top: 30px;
+    max-width: 800px;
+    min-width: 400px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+
+  .contact-me {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .contact-bg2 {
+    margin-left: 266px;
+    max-height: 500px;
+    margin-bottom: 39rem;
+  }
+
+  .view-btn {
+    margin-right: 39rem;
+    width: 25%;
+  }
+
+  .main-text.contact-text h1 {
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+    width: 385px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 52px;
+    margin-right: 27rem;
+    font-size: 31px;
+    margin-bottom: 49rem;
+  }
+
+  .form {
+    width: 100%;
+    min-width: 300px;
+    margin-bottom: 49rem;
+  }
```

Comment:
- [ ] I have noticed that the form area may be among one causing the horizontal scroll in your desktop version portifolio. Remember, The experience of scrolling horizontally can makes the screen judder and create a headache inducing feeling. Therefore I suggest that you fix horizontal scroll effect in all the desktop screen sizes.
Your design 😎 . Please disregard the background colors.
![image](https://github.com/AlyAtalla/AlyAtalla/assets/99394374/f12c7906-5629-4514-92fd-9afef14882dd)